routes/signUpRoute.js

```diff
@@ -0,0 +1,18 @@
+const router = require('express').Router()
```

Comment:
@AwaleRohin keep the auth route in single file. these are open routes we don't need authorization for those routes